src/examples/install-helm-chart.yml

```diff
@@ -4,7 +4,7 @@ usage:
   version: 2.1
 
   orbs:
-    aws-eks: circleci/aws-eks@2.0
+    aws-eks: circleci/aws-eks@2.1
```

Comment:
is this correct? it is not consistent across examples

Reply:
Yeah that's correct.  The latest version is version 2.1

Reply:
https://circleci.com/developer/orbs/orb/circleci/aws-eks